installer/datafiles/base_container.data

```diff
@@ -125,8 +125,9 @@ MAINTAINER:              'Microsoft Corporation'
 
 
 /opt/microsoft/omsagent/plugin/health/aggregate_monitor.rb;                                         source/code/plugin/health/aggregate_monitor.rb; 644; root; root
-/opt/microsoft/omsagent/plugin/health/aggregate_monitor_instance_id_labels.rb;                      source/code/plugin/health/aggregate_monitor_instance_id_labels.rb; 644; root; root
+/opt/microsoft/omsagent/plugin/health/agg_monitor_id_labels.rb;                      		    source/code/plugin/health/agg_monitor_id_labels.rb; 644; root; root
```

Comment:
is this is what was causing the issue, this does not look that long. Are we sure the root cause is this ?

Reply:
yeah. 100% positive. only change was the change to the filename and the agent started working on newer kernels